integration_tests/src/main/python/hashing_test.py

```diff
@@ -31,36 +29,58 @@
     timestamp_gen,
     decimal_gen_32bit,
     decimal_gen_64bit,
-    decimal_gen_128bit]
-if not is_before_spark_320():
-    _xxhash_gens += [float_gen, double_gen]
+    decimal_gen_128bit,
```

Comment:
Why is the spark version limit removed ?

Reply:
We already removed all the 31x shims.